libkbfs/file_data.go

```diff
@@ -65,13 +65,21 @@ type parentBlockAndChildIndex struct {
 	childIndex int
 }
 
+func (pbci parentBlockAndChildIndex) childIPtr() IndirectFilePtr {
+	return pbci.pblock.IPtrs[pbci.childIndex]
+}
+
+func (fd *fileData) rootBlockPointer() BlockPointer {
```

Comment:
I just realized you might be talking about making a new PR that is merged _before_ this one (I was assuming it would be after, or independent).  I think that could make sense.